CONTRIBUTING.md

```diff
@@ -8,8 +8,13 @@ Contributors Guide
 
 Discuss contributions to DC/OS SDK using GitHub [issues](https://github.com/mesosphere/dcos-commons/issues) and make contributions with GitHub [pull requests](https://github.com/mesosphere/dcos-commons/pulls).
 
-### Proposing Features
-If you would like to propose a new feature or a significant change, first open an issue to discuss it.  Project shepherds will provide feedback on the proposed change in terms of feasibility, design, and implementation guidance.  While all proposals are welcome, those that conflict with our roadmap or design principles may not be accepted.
+### Bugs / Feature Requests
+Think you’ve found a bug? Want to see a new feature? Please open a case in our issue management tool, JIRA:
+* Login to the [Mesosphere JIRA](jira.mesosphere.com).  You will need a github or google account to use this service.  
```

Comment:
GitHub + Google capitalized?

Reply:
Please reinstate the sentences about "shepherds" and "roadmap".